src/main/scala/mission/impossibl/bots/orchestrator/GarbageOrchestrator.scala

```diff
@@ -42,6 +48,12 @@ object GarbageOrchestrator {
                 orchestrator(instance, state.copy(auctionsInProgress = state.auctionsInProgress.removed(auctionId)))
               case None => Behaviors.same
             }
+
+          case GarbageScore(sourceId: Int, garbage_score: Int) =>
+            context.log.info("Waste source with id {} got score {}", sourceId, garbage_score)
+            val waste_source = state.wasteSources.get(sourceId)
```

Comment:
get zwraca option, trzeba zmapować 
state.wasteSources.get(sourceId).flatMap( _ ! GarbageScoreSummary(...))

Reply:
I jeszcze uwaga co do logiki - tu mamy w diagramie działania systemu info o tym że te score jest jakoś agregowane po stronie orkiestratora

Reply:
tutaj nadal coś głupieje,

Jeśli chodzi o agregacje to można by jakoś uśredniać poprzedni wynik i aktualny (ale to by wymagało zaciągania danych z source, więc nie wiem czy chcemy to dodawać)

Reply:
Sorry map nie flatMap, źle ci wcześniej napisałam. 
Jeśli chodzi o agregację to imo orkiestrator może przechowywać mapę score'ów i przesyłać nie z każdym nowym ale cyklicznie po prostu, a source utrzymuje np 4 ostatnie scory tylko

Reply:
poprawione tak, że się kompiluje. Dzisiaj nie zdążę tej funkcjonalności dodatkowej zrobić. Może jak będę miał czas to w piątek siądę